specs/brew/homebrew_tap_repository.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Tap a Homebrew tap repository from GitHub
```

Comment:
Oop! Overlooked that, thanks for pointing that out.

Reply:
Do you think it should be `formula_repository` or `formulae_repository` for the name?

Reply:
Slight preference towards `formula_repository` since that's the wording brew uses themselves:
<img width="1138" alt="image" src="https://user-images.githubusercontent.com/4110292/164026379-7684041c-c11a-446b-a258-07a8eaa56743.png">